### DISCUSSION TITLE
Change exiting out of editor button placement

### DISCUSSION THREAD

**Discussion by CaptainBasculin** [ppy/osu#17363] [Ideas]
In my first days of trying the osu!lazer's map editor, I've had multiple instances where I accidentally quit out of editor with nothing saved. At my first try of editor, I was like "is this some sort of editor glitch?" but over time I've understood why. On the stable editor, ESC is used as the unselect character; and even in an accidental missclick to esc, the escape screen buttons are small enough that accidentally hitting them never happens. 
![resim](https://user-images.githubusercontent.com/48386548/159160028-6310628d-eef1-4c94-9633-042395cacb31.png)

However on lazer editor, ESC doesn't unselect object and leads directly to escape screen, which covers a noticable portion of the editor, which made me accidentally hit it a lot. 
![osu_2022-03-20_14-01-26](https://user-images.githubusercontent.com/48386548/159160160-a0751d81-6451-4b4e-95cc-00ec0a83ba83.jpg)

If it were left to me, moving the buttons to outside of mapping grid, or switching to small "Yes / No / Cancel" buttons would fit better. But, I'm fine with anything that doesn't lead to accidental clicks.

**Comment by bdach** [comment]
moving the dialog is not really an option unless all dialogs game-wide were to be moved. all dialogs in the game are positioned in the same manner, making an exception for editor would be weird.

maybe the press esc to deselect behaviour should just be brought back, dunno.

**Reply by peppy** [reply]
I'd say the first escape deselecting may be fine to add back.

I also think there's been previous discussion of making "destructive" buttons require a short hold to activate, which in cases like this especially makes sense.

**Reply by peppy** [reply]
I've added the hold-to-confirm via #17384, should be a good starting point.
